index.html

```diff
@@ -104,102 +132,104 @@ <h4>Project names goes here</h4>
       <div class="full-section-div">
         <div id="img-div5" class="imj-project-section"><img id="full-section-div6" src="./images/desktop6.png" alt=""></div>
         <div id="list-of-project6" class="list-of-project">
-         <h4>Project names goes here </h4>
+         <h4>Project names   here </h4>
          <ul>
               <li class="project-list-1">HTML/CSS</li>
               <li class="project-list-2">Ruby on Rails</li>
               <li class="project-list-3">Javascricpt</li>
+              <li class="project-list-4">+1</li>
           </ul>
           <div class="project-link"><a href="#">See this project</a>
            <img class="img-list" src="./images/arrow-right.png" alt="arrow"></div>
            <img id="img-geo" src="./images/image-geometry_5.png" alt="my last img">
         </div>
       </div>     
-     </div>
+     </div>    
   </section>
   <section id="about-section">
-      <div class="about-section-container">
-          <div class="about-section-right">
-               <h2>About</h2>
-               <div id="about-sction-div">
-                   <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. 
-                      Labore praesentium, ipsa necessitatibus maioresquaeiusto eveniet
-                      asperiores at alias debitis deserunt excepturi, esse sequi.</p>
-                    <button type="submit">Get my cv   <img id="download" src="./images/arrow-right.png" alt=""></button>
-              </div>
-            </div>
-          <div class="about-section-left">
-               <div id="about-section-list">
-                     <div id="language">
-                        <h5>Languages</h5>
-                        <ul >
-                              <li>Javascricpt(ES5/ES6)</li>
-                              <li>Ruby</li>
-                              <li>HTML</li>
-                              <li>CSS</li>
-                        </ul>
-              </div>
-              <div id="framework">
-                   <h5>Frameworks</h5>
-                  <ul>
-                  <li>Bootsrap</li>
-                  <li>Ruby on Rails</li>
-                  <li>Rspec</li>
-                  <li>Capibarra</li>
-                  <li>Selenium</li>
-                  </ul>
-              </div>
-              <div id="skills">
-                  <h5>Skills</h5>
-                  <ul>
-                  <li>Codekit</li>
-                  <li>Codepen</li>
-                  <li>Github</li>
-                  <li>Gitlab</li>
-                  <li>Terminal</li>
-                 </ul>
-              </div>
-              <img id="teal-img" src="./images/image_dots_with_teal.png" alt="">
-            </div>
-          </div>
+    <div class="about-section-container">
```

Comment:
- [ ] [OPTIONAL] Your About section doesn't have the proper margins, and font family. Kindly stick to the Figma design as much as possible.

|Your design|Figma design|
|-|-|
|![Screenshot_20220611_051730](https://user-images.githubusercontent.com/25227022/173172192-af15aa61-24e7-4870-b043-00797e5718e6.png)|![172935865-b45fa15c-a697-443c-aee3-ed78b17cfd7b](https://user-images.githubusercontent.com/25227022/173172240-6adc9ef9-1a16-4882-badf-9a8fcedf45c9.png)|